utils/check_repo.py

```diff
@@ -259,6 +262,7 @@
     "InstructBlipQFormerModel",
     "InstructBlipVideoVisionModel",
     "InstructBlipVideoQFormerModel",
+    "Kosmos2_5ForConditionalGeneration",
```

Comment:
This should be removed

Reply:
@NielsRogge  mentioned not to add this model to any auto mapping. Not sure if his comment is still relevant. If so, we need to keep this entry here.

@NielsRogge Could you check again?

Reply:
It can now be added to the image-text-to-text [mapping](https://github.com/huggingface/transformers/blob/8b3b9b48fcd6bc06bd9c576f1b09266d577db257/src/transformers/models/auto/modeling_auto.py#L764), I assume?